xbmc/pvr/windows/GUIWindowPVRTimersBase.cpp

```diff
@@ -372,12 +374,7 @@ bool CGUIWindowPVRTimersBase::ActionShowTimer(CFileItem *item)
   }
   else
   {
-    const CPVRTimerInfoTagPtr tag(item->GetPVRTimerInfoTag());
-    if (ShowTimerSettings(tag) && !tag->GetTimerType()->IsReadOnly())
-    {
-      /* Update timer on pvr backend */
-      bReturn = g_PVRTimers->UpdateTimer(tag);
-    }
+    bReturn = EditTimer(item);
```

Comment:
Note to self/reviewer: this fixes a crash after changing timer type, because I forgot to prpare this code for the new feature. Best is to use EditTimer here as it does exactly the right thing and we have no code duplication.